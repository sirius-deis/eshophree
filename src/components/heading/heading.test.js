import { render, screen } from "@testing-library/react";
import Heading from "./heading";

describe(" Heading component", function() {
  it("should match snapshot", function() {
    const { container } = render(<Heading>Hello World</Heading>);
    expect(container).toMatchSnapshot();
  });
  it("should render with different text", function() {
    render(<Heading>Hello Universe</Heading>);
    expect(screen.getByText("Hello Universe")).toBeInTheDocument();
  });
})