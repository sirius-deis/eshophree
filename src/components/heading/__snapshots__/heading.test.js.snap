// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Heading component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB kPHljY"
  >
    Hello World
  </div>
</div>
`;
